talksapp/README.md

```diff
@@ -6,7 +6,7 @@ This directory contains the source for [go-talks.appspot.com](http://go-talks.ap
 Development Environment Setup
 -----------------------------
 
-- Copy config.go.template to config.go and edit the file as described in the comments.
+- Edit env\_variables in the `app.yaml` file as described in the comments.
```

Comment:
have this say "make a copy of app.yaml named prod.yaml and put in the authentication data". we don't want these values accidentally committed and leaked